### PR TITLE
Display product integration

### PR DESCRIPTION
**Pull request by RichardSimpson235**
Display product feature merge conflicts resolved.

### CODE REVIEW

File: dart-cart-client/src/App.test.tsx [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+//     rest.get('/Display', (req, res, ctx) => {
+//       return res(ctx.json({greeting: 'Make Tanner Do it pls.'}))
+//     }),
+//   )
+
+//   fireEvent.click(screen.getByText('Chosen Shop Product'))
+
+// });
+
+
```
Comment (RichardSimpson235) [inline dart-cart-client/src/App.test.tsx:22]: Need to delete

File: dart-cart-client/src/__Tests/App.test.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  {
+    ////////////////////////////////////////// G E T ____ B A N N E D //////////////////////////////////////////////////////////}
+  }
+  {
+    ////////////////////////////////////////// G E T ____ B A N N E D //////////////////////////////////////////////////////////}
+  }
+  {
+    ////////////////////////////////////////// G E T ____ B A N N E D //////////////////////////////////////////////////////////}
+  }
+});
```
Comment (RichardSimpson235) [inline dart-cart-client/src/__Tests/App.test.tsx:32]: Need to delete

File: dart-cart-client/src/services/APIQuery.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//         }),
+//         getAllShopProducts: builder.query({
+//             query: () => "/ShopProduct"
+//         })
+//     })
+// })
+
+// export const { useGetShopProductByIdQuery, useGetAllShopProductsQuery } = StoreAPI
+// useGetShopProductByIdQuery AND useGetAllShopProductsQuery are @@@@@@@ A U T O - G E N E R A T E D @@@@@@@@ functions that come directly from
+// getShopProductById AND getAllShopProducts ENDPOINT names respectively. Their format is dynamic camelcode based on what you call the ENDPOINT builders
```
Comment (RichardSimpson235) [inline dart-cart-client/src/services/APIQuery.ts:27]: Need to delete

**Review by RichardSimpson235 — COMMENTED** [review]
Need to remove noted files.

**Review by redn6gx — APPROVED** [review]
Looks good

**Pull request merged**